R/ecdc.R

```diff
@@ -91,10 +81,9 @@ get_ecdc_cases <- function (countries = NULL){
 format_ecdc_data <- function(data = NULL) {
```

Comment:
I am not sure if we need to keep this function - only we use it and we use it twice. Perhaps we should be just doing this cleaning in the script?

Reply:
Removed. The cleaning here is specific for EpiNow (eg renaming "region" instead of country) so not adding to main script.